exchange/exchange-ps/exchange/Set-RetentionCompliancePolicy.md

```diff
@@ -88,6 +88,8 @@ Set-RetentionCompliancePolicy [-Identity] <PolicyIdParameter>
 ## DESCRIPTION
 To use this cmdlet in Security & Compliance Center PowerShell, you need to be assigned permissions. For more information, see [Permissions in the Microsoft 365 compliance center](https://docs.microsoft.com/microsoft-365/compliance/microsoft-365-compliance-center-permissions).
 
+**Note**: Please avoid using a piped Foreach-Object when adding or removing scope Locations : "Value1","Value2",..."ValueN" | Foreach-Object { Set-RetentionCompliancePolicy -Identity "Regulation 123 Compliance" -RemoveExchangeLocation $_  }
```

Comment:
```suggestion
**Note**: Avoid using a piped Foreach-Object when adding or removing scope Locations : "Value1","Value2",..."ValueN" | Foreach-Object { Set-RetentionCompliancePolicy -Identity "Regulation 123 Compliance" -RemoveExchangeLocation $_  }
```

Reply:
@chrisda  is going with "Don't use a piped Foreach-Object" for the other PRs